squad-server/index.js

```diff
@@ -456,6 +509,9 @@ export default class SquadServer extends EventEmitter {
             connectorConfig.filter,
             connectorConfig.activeLayerFilter
           );
+        } else if (option.connector === 'remoteAdminLists') {
+          Logger.verbose('SquadServer', 1, `Starting remoteAdminList connector...`);
+          connectors[connectorName] = await server.setupAdminList(connectorConfig);
```

Comment:
The admin list is information about the server so can go into the `SquadServer` class rather than being a connector. Connectors should only be used for stuff that's not directly related to the server, e.g. Discord, MySQL or subsets of Squad maps (i.e. rotations/pools).